resources/widgets.py

```diff
@@ -70,7 +70,7 @@ def get(self):
                     "desc": 'In Progress'
                 },
                 {
-                    "stat": TicketModel.find_count_by_age_status("In Progress", 10080),
+                    "stat": TicketModel.find_count_by_update_status("In Progress", 10080),
```

Comment:
Looks like we're tracking state with hard-code strings. FYI: Created issue codeforpdx/dwellingly-app/issues/404 to address this.